src/Automation/Automation/help/New-AzAutomationUpdateManagementAzureQuery.md

```diff
@@ -0,0 +1,161 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Automation.dll-Help.xml
+Module Name: Az.Automation
+online version:
```

Comment:
@jemex you will need to add the correct URL for this cmdlet as it would appear on docs. Here's the message from the `ValidateHelpIssues.csv` file in the build artifacts:

```
Online version in the header of the file is incorrect. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/az.automation/new-azautomationupdatemanagementazurequery
```

Reply:
@cormacpayne Thank you for the tip, correct url is added